### PR TITLE
Add doublehead_rcnn detection module

### PR DESCRIPTION
**Pull request by 315386775**
Double Head Faster RCNN model from the paper. 
(2019). Rethinking Classification and Localization for Object Detection.
Models of VOC dataset are evaluated with native resolutions with shorter side >= 600 but longer side <= 1000 without changing aspect ratios.

| detection module | map | 
| ----- | ----- | 
| faster_rcnn_resnet50_v1b_voc | 78.3 |
| doublehead_rcnn_resnet50_v1b_voc| 81.0 |

### CODE REVIEW

**Comment by 315386775** [comment]
@bryanyzhu i have upload the bash/models/logs to dropbox. Link:
https://www.dropbox.com/sh/6fol4qaxyd0fabi/AAAmmOFIUGAh-p04AbuzZXy5a?dl=0

i will pull more detection modules. thanks.

File: README.md [hunk 1]
```diff
@@ -190,3 +190,5 @@ If you feel our code or models helps in your research, kindly cite our papers:
   year={2020}
 }
 ```
+
```
Comment (bryanyzhu) [inline README.md:193]: Please remove these two lines from README, thank you.

File: gluoncv/model_zoo/rcnn/faster_rcnn/doublehead_rcnn.py [hunk 2]
```diff
@@ -0,0 +1,642 @@
+"""
+Double Head Faster RCNN Model.
```
Comment (bryanyzhu) [inline gluoncv/model_zoo/rcnn/faster_rcnn/doublehead_rcnn.py:2]: Can you add more information, like the author information and license information? Thank you.

**Review by bryanyzhu — CHANGES_REQUESTED** [review]
Awesome PR. Please fix two comments and I will merge it.

File: gluoncv/model_zoo/rcnn/faster_rcnn/doublehead_rcnn.py (hunk 2 above)
Comment (315386775) [reply gluoncv/model_zoo/rcnn/faster_rcnn/doublehead_rcnn.py]: @bryanyzhu thank you.  i have modified the related information. 
i have another question with workflow error in https://github.com/dmlc/gluon-cv/pull/1727
thanks.

Comment (315386775) [reply gluoncv/model_zoo/rcnn/faster_rcnn/doublehead_rcnn.py]: @bryanyzhu how to fix this error. thanks.

======================================================================
ERROR: tests.unittests.test_data_datasets.test_lst_detection
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/miniconda/envs/gluon_cv_py3_test/lib/python3.6/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/runner/work/gluon-cv/gluon-cv/tests/unittests/test_data_datasets.py", line 111, in test_lst_detection
    path='dog.jpg')
  File "/Users/runner/work/gluon-cv/gluon-cv/gluoncv/utils/download.py", line 69, in download
    raise RuntimeError("Failed downloading url %s"%url)
RuntimeError: Failed downloading url https://github.com/dmlc/web-data/blob/master/gluoncv/datasets/dog.jpg?raw=true
-------------------- >> begin captured stdout << ---------------------
Downloading dog.jpg from https://github.com/dmlc/web-data/blob/master/gluoncv/datasets/dog.jpg?raw=true...

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): github.com:443
urllib3.connectionpool: DEBUG: https://github.com:443 "GET /dmlc/web-data/blob/master/gluoncv/datasets/dog.jpg?raw=true HTTP/1.1" 429 1474
--------------------- >> end captured logging << ---------------------

**Pull request merged**